guests.json

```diff
@@ -76,5 +76,10 @@
   {
     "hd": "hd/partycage.gif",
     "name": "Party Cage"
+  },
+  {
+    "hd": "guests/hd/batparrot.gif",
```

Comment:
Can you make it just `hd/batparrot.gif`?  Sorry!

Reply:
Oh good lord! 
Don't apologise...
🤦